packages/rocketchat-lib/server/functions/getRoomByNameOrIdWithOptionToJoin.js

```diff
@@ -1,7 +1,11 @@
 /* globals RocketChat */
 import _ from 'underscore';
 
+<<<<<<< HEAD
```

Comment:
Here's a merge conflict that didn't got resolved

Reply:
Done

Reply:
@auterium   -- you approved the commit to close this out an hour ago below so seems we could click this conversation as resolved as well   auterium requested changes a day ago

"5fef911  Merged Marc's 'teams' changes without adjustment to getRoomByNameOrId…  …
"